media/starboard/starboard_renderer.cc

```diff
@@ -524,10 +591,20 @@ void StarboardRenderer::OnDemuxerStreamRead(
     if (stream == audio_stream_) {
       DCHECK(audio_read_in_progress_);
       audio_read_in_progress_ = false;
+      for (const auto& buffer : buffers) {
+        last_audio_sample_interval_ =
+            buffer->timestamp() - timestamp_of_last_written_audio_;
```

Comment:
@osagie98 this hits DCHECK() on devel build when a video is playing to the end, due to calling timestamp() on an EOS buffer.